parm/aero/variational/3dvar_gfs_aero.yaml

```diff
@@ -2,7 +2,9 @@ cost function:
   cost type: 3D-Var
   window begin: '$(AERO_WINDOW_BEGIN)'
   window length: $(AERO_WINDOW_LENGTH)
-  analysis variables: &3dvars [ua,va,t,ps,sphum,ice_wat,liq_wat,o3mr]
+  analysis variables: &3dvars [sulf,bc1,bc2,oc1,oc2,
```

Comment:
We need to talk about this offline, I fear we have work ahead of us in UFO/FV3-JEDI to support the UFS coupled model with aerosols. So let's hold off on merging this until I know for sure.

Reply:
Does `sulf` work? Or does it fail on that?

Reply:
Also it looks like this PR does *not* add in increment write...

Reply:
@andytangborn I just realized that you did not push a new commit to this PR... it is from 10 days ago.